src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+namespace detail {
+// @{
+/*!
+ * \brief Computes the expression needed to set boundary conditions on dt<VZero>
+ *
+ * \details Computes the correction to the characteristic projected
+ * time-derivatives of the fundamental variables corresponding to \f$u^0\f$:
+ *
+ * \f{align}
+ * \partial_t u^{0}_{iab} = \partial_t u^{0}_{iab} + v_{0} n^j C_{jiab}
+ * \f}
+ *
+ * where \f$n^i\f$ is the local unit normal to the external boundary,
+ * and \f$C_{ijab} = \partial_i\Phi_{jab} - \partial_j\Phi_{iab}\f$ is the
+ * four-index constraint.
+ */
+template <size_t VolumeDim, typename DataType>
+void set_dt_v_zero_constraint_preserving(
```

Comment:
again, not a change request: this can also be written fairly easily as a correction. I worked out what the corrections to the GH evolved var time derivatives are:
```
\begin{align}
  \label{eq:fosh gh constraing preserving bc g}
  \partial_t g_{ab}&\to \partial_t g_{ab} + \lambda_{w^g}s^i\mathcal{C}_{iab},
  \\
  \label{eq:fosh gh constraing preserving bc pi}
  \partial_t \Pi_{ab}&\to \partial_t \Pi_{ab}
                       +\frac{1}{2}\lambda_{w^-}
                       \left[B_{ab}^C+B_{ab}^P+B_{ab}^G\right]
                       +\gamma_2 \lambda_{w^g}s^i\mathcal{C}_{iab}, \\
  \label{eq:fosh gh constraing preserving bc phi}
  \partial_t \Phi_{iab}&\to\partial_t \Phi_{iab}
                         -\frac{s_i}{2}\lambda_{w^-}
                         \left[B_{ab}^C+B_{ab}^P+B_{ab}^G\right]
                         +\lambda_{w^0}s^iP^j{}_k\mathcal{C}_{ijab}.
\end{align}
```
(just to confused everyone more, `w` are the char fields -_-). The `B_{ab}` terms are the boundary conditions (`C`=constraint preserving, `G`=gauge, `P`=physical for superscripts).

Reply:
agreed that just having corrections implemented saves computation, and have made this change